dist-git.spec

```diff
@@ -85,70 +86,37 @@ cd -
 getent group packager > /dev/null || \
     groupadd -r packager
 
-getent group gen-acls > /dev/null || \
-    groupadd -r gen-acls
-
-getent passwd gen-acls > /dev/null || \
-    useradd -r -g gen-acls -G packager -s /bin/bash \
-            -d %{_sharedstatedir}/dist-git/git gen-acls
-
 
 %install
 # ------------------------------------------------------------------------------
-# /usr/share/ .... static files
+# /usr/local/bin ........... scripts
```

Comment:
/usr/local should not be used in proper package. /usr/share is the right place for static files.